docs/guide/rest-filtering-collections.md

```diff
@@ -0,0 +1,191 @@
+
+
+## Filter Control Keywords <span id="filter-control-keywords"></span>
+
+The default list of allowed filter control keywords is as the following:
+
+| filter control | translates to |
+|:--------------:|:-------------:|
+|     `and`      |     `AND`     |
+|      `or`      |     `OR`      |
+|     `not`      |     `NOT`     |
+|      `lt`      |      `<`      |
+|      `gt`      |      `>`      |
+|     `lte`      |     `<=`      |
+|     `gte`      |     `>=`      |
+|      `eq`      |      `=`      |
+|     `neq`      |     `!=`      |
+|      `in`      |     `IN`      |
+|     `nin`      |   `NOT IN`    |
+|     `like`     |    `LIKE`     |
+
+You can expand that list by expanding option [[yii\data\DataFilter::$filterControls]], for example you could provide
+several keywords for the same filter build key, creating multiple aliases like:
+
+```php
+[
+    'eq' => '=',
+    '=' => '=',
+    '==' => '=',
+    '===' => '=',
+    // ...
+]
+```
+
+Keep in mind that any unspecified keyword will not be recognized as a filter control and will be treated as an attribute 
+name - you should avoid conflicts between control keywords and attribute names (for example: in case you have control 
+keyword `like` and an attribute named `like`, specifying condition for such attribute will be impossible).
+
+> Note: while specifying filter controls take actual data exchange format, which your API uses, in mind.
+  Make sure each specified control keyword is valid for the format. For example, in XML tag name can start
+  only with a letter character, thus controls like `>`, `=`, or `$gt` will break the XML schema.
+
+> Note: When adding new filter control word make sure to check whether you need also to update 
+  [[yii\data\DataFilter::$conditionValidators]] and/or [[yii\data\DataFilter::$operatorTypes]] in order to achieve
+  expected query result based on the complication of the operator and the way it should work.
+
+
+## Handling The Null Values <span id="handling-the-null-values"></span>
+
+While it is easy to use `null` inside the JSON statement, it is not possible to sent it using the GET query without 
+confusing the literal `null` with the string `"null"`. Since 2.0.40 you can use [[yii\data\DataFilter::$nullValue]] 
+option to configure the word that will be used as a replacement for literal `null` (by default it's `"NULL"`).
+
+
+## Aliasing Attributes <span id="aliasing-attributes"></span>
+
+Whether you want to alias the attribute with another name or to filter the joined DB table you can use
+[[yii\data\DataFilter::$attributeMap]] to set the map of aliases:
+
+```php
+[
+    'carPart' => 'car_part', // carPart will be used to filter car_part property
+    'authorName' => '{{author}}.[[name]]', // authorName will be used to filter name property of joined author table
+]
+```
+
+## Configuring Filters For ActiveController <span id="configuring-filters-for-activecontroller"></span>
```

Comment:
```suggestion
## Configuring Filters For `ActiveController` <span id="configuring-filters-for-activecontroller"></span>
```

Reply:
Or maybe use a link. :thinking:

Reply:
I'm not sure how it looks rendered in the guide, also the link is provided in the very first sentence of that paragraph.

Reply:
Highlighting example can be found here. https://www.yiiframework.com/doc/guide/2.0/en/caching-http#last-modified Not sure about link too. It's not critical. :ok_hand: